plugins/search-backend-module-elasticsearch/src/engines/ElasticSearchSearchEngine.ts

```diff
@@ -140,6 +140,8 @@ export class ElasticSearchSearchEngine implements SearchEngine {
     term,
```

Comment:
The elastic search part is not really tested, I leave that up for somebody else 😉